src/Generator.Bind/LicenseResource.cs

```diff
@@ -0,0 +1,32 @@
+using System.IO;
```

Comment:
Missing file header - use the template from the style guide.

Reply:
Will fix it.